assets/js/container/register-maker/register-maker.js

```diff
@@ -285,11 +297,28 @@ class RegisterMaker extends React.Component {
     if (this.context.user && this.context.user.id) {
       // todo redirect to home?
     }
+
+    this.getCountryList("de");
   }
 
-  onSubmit = (data) => {
-    console.log(data);
+  getCountryList(lang) {
+    const url = `https://restcountries.eu/rest/v2/all?fields=alpha2Code;name;translations`;
```

Comment:
we should consider storing the country list locally since fetching it remotely would introduce another point of failure...

Reply:
done. I put the file as static to /assets/meta/country-codes.json. so it can be fetched from our repository